prisma/schema.prisma

```diff
@@ -36,11 +36,19 @@ model booking_status {
 }
 
 model prospects {
-  id       Int        @id @default(autoincrement())
+  id       Int            @id @default(autoincrement())
```

Comment:
I'm pretty sure you have to generate a migration folder when changing the database schema.